core/tauri-utils/src/config.rs

```diff
@@ -264,6 +264,13 @@ pub struct WindowsConfig {
   /// The fixed version can be downloaded [on the official website](https://developer.microsoft.com/en-us/microsoft-edge/webview2/#download-section).
   /// The `.cab` file must be extracted to a folder and this folder path must be defined on this field.
   pub webview_fixed_runtime_path: Option<PathBuf>,
+  /// Validates a second app installation, blocking the user from installing an older version if set to `false`.
+  ///
+  /// For instance, if `1.2.1` is installed, the user won't be able to install app version `1.2.0` or `1.1.5`.
+  ///
+  /// The default value of this flag is `true`.
+  #[serde(default)]
```

Comment:
But isn't `serde[default]` for bools `false`?

Reply:
nice catch, thanks

Reply:
that's a problem with configuration flags that defaults to true :( way easier if it defaults to false